samples: lwm2m_carrier: decrease http fragment size (#22121)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Cellular samples
----------------

> * LwM2M carrier library: Decreased the fragment size on cellular samples which include the LwM2M carrier library.
>   This is done to ensure FOTA issued from Verizon LwM2M servers has enough room for HTTP headers.
> 
>   * :ref:`slm_description`
> 
>   * :ref:`http_application_update_sample`
> 
>   * :ref:`lwm2m_carrier`
> 
>   * :ref:`modem_shell_application`

Reviewer: ```suggestion
* Decreased the fragment size on the following cellular samples that include the :ref:`liblwm2m_carrier_readme` library.

  * :ref:`slm_description`
  * :ref:`http_application_update_sample`
  * :ref:`lwm2m_carrier`
  * :ref:`modem_shell_application`

    This is done to ensure that FOTA issued from Verizon LwM2M servers has enough room for HTTP headers.
```

Reviewer 2: @divipillai, fyi, to be moved later to the release notes.
